src/renderer/components/dialogs/CreateChat/index.tsx

```diff
@@ -647,37 +647,61 @@ const useCreateGroup = (
   const accountId = selectedAccountId()
   const { selectChat } = useChat()
 
-  const createGroup = useCallback(async () => {
-    const isVerified = await areAllContactsVerified(accountId, groupMembers)
+  const createGroup = useCallback(
+    async (groupTemplateId?: number | null) => {
+      let finalGroupName = groupName
 
-    const chatId = await BackendRemote.rpc.createGroupChat(
-      accountId,
-      groupName,
-      isVerified
-    )
+      if (groupTemplateId) {
+        const chat = await BackendRemote.rpc.getFullChatById(
+          accountId,
+          groupTemplateId
+        )
 
-    if (groupImage && groupImage !== '') {
-      await BackendRemote.rpc.setChatProfileImage(accountId, chatId, groupImage)
-    }
+        groupMembers.push(...chat.contactIds)
```

Comment:
Is it OK to mutate the parameter? Would it not add the same members twice to the same array if you cloned the group twice?
I'd just clone the array.